client/src/common/components/Home.tsx

```diff
@@ -3,164 +3,198 @@ import { Link } from 'react-router-dom';
-								consequatur voluptatibus aut repudiandae? Hic repudiandae eos
-								eveniet amet sit architecto nostrum vitae ipsam.
-							</p>
-						</AccordionItemPanel>
-					</AccordionItem>
-					<AccordionItem className="p-5">
-						<AccordionItemHeading>
-							<AccordionItemButton>Why you need Senpai ?</AccordionItemButton>
-						</AccordionItemHeading>
-						<AccordionItemPanel>
-							<p>
-								dolorum! Iste assumenda molestias, rerum provident eos
-								consequatur voluptatibus aut repudiandae? Hic repudiandae eos
-								eveniet amet sit architecto nostrum vitae ipsam. dolorum! Iste
-								assumenda molestias, rerum provident eos consequatur
-								voluptatibus aut repudiandae? Hic repudiandae eos eveniet amet
-								sit architecto nostrum vitae ipsam.
-							</p>
-						</AccordionItemPanel>
-					</AccordionItem>
-					<AccordionItem className="p-5">
-						<AccordionItemHeading>
-							<AccordionItemButton>How does it work ?</AccordionItemButton>
-						</AccordionItemHeading>
-						<AccordionItemPanel>
-							<p>
-								Exercitation in fugiat est ut ad ea cupidatat ut in cupidatat
-								occaecat ut occaecat consequat est minim minim esse tempor
-								laborum consequat esse adipisicing eu reprehenderit enim.
-							</p>
-						</AccordionItemPanel>
-					</AccordionItem>
-					<AccordionItem className="p-5">
-						<AccordionItemHeading>
-							<AccordionItemButton>
-								What you can do for Kohais ?
-							</AccordionItemButton>
-						</AccordionItemHeading>
-						<AccordionItemPanel>
-							<p>
-								Exercitation in fugiat est ut ad ea cupidatat ut in cupidatat
-								occaecat ut occaecat consequat est minim minim esse tempor
-								laborum consequat esse adipisicing eu reprehenderit enim.
-								dolorum! Iste assumenda molestias, rerum provident eos
-								consequatur voluptatibus aut repudiandae? Hic repudiandae eos
-								eveniet amet sit architecto nostrum vitae ipsam.
-							</p>
-						</AccordionItemPanel>
-					</AccordionItem>
-				</Accordion>
-			</section>
-			<section className="p-10">
-				<h2 className="text-2xl md:text-5xl font-bold text-center">Contact</h2>
-				<div className="md:flex md:p-10 p-10 md: grid justify-items-stretch">
-					<img
-						className="md:w-3/6 w-3/4 md:p-20 justify-self-center"
-						src="./assets/contact.svg"
-						alt="contact"
-					/>
+            >
+              Join Now
+            </button>
+            {/* ↓ for mobile version */}
+            <button
+              className="cursor-pointer 
+								mt-5 
+								py-2.5
+								w-56
+								text-primary_bg_color 
+								bg-primary_title_color 
+								rounded-full 
+								font-black
+								hover:bg-secondary_bg_color 
+								mobile_m:hidden
+								"
+            >
+              Join Now
+            </button>
+          </Link>
+        </div>
+        <img
+          className="w-7/12 tablet_md_max:hidden"
```

Comment:
```suggestion
          className="w-7/12 max-w-2xl tablet_md_max:hidden"
```
The image looks too big for the desktop screen, so added the maximum size of the image.